media/css/pocket/components/home.scss

```diff
@@ -18,6 +18,18 @@ $image-path: '/media/img/pocket';
     .pocket-homepage-header {
         padding-top: 0;
 
+        .mzp-l-split-media-overflow {
+            width: 145%;
+            transform: translateX(-40%);
+            left: 40%;
+
+            @media #{$mq-md} {
+                width: 100%;
+                transform: unset;
+                left: unset;
+            }
+        }
```

Comment:
@maureenlholland referencing your [image overflow comment](https://github.com/mozilla/bedrock/pull/11592#discussion_r870162624), instead of using your suggestion I just used the styles you had to make the image overflow in the `pocket-marketing-pages` codebase to make it work the way we want. Hope this is good enough?